package.json

```diff
@@ -23,7 +23,7 @@
     "@octokit/plugin-paginate-graphql": "^5.2.4",
     "ansi-styles": "^6.2.1",
     "esbuild": "^0.25.4",
-    "temporal-polyfill": "^0.2.5",
+    "temporal-polyfill": "^0.3.0",
```

Comment:
It might be a spec change for this library. 

https://github.com/fullcalendar/temporal-polyfill/blob/HEAD/CHANGELOG.md#v030-2025-03-28

Reply:
There is no silent changes, so CI passed means I don't need any migration with this updates.